SaaSHelperFunctions.ps1

```diff
@@ -0,0 +1,30 @@
+﻿function Create-SaasUrl {
+    Param(
+        [Parameter(Mandatory = $true)]
+        [Hashtable] $bcAuthContext,
+        [string] $applicationFamily = ('', 'BusinessCentral')[$PSBoundParameters.ContainsKey('environment')],
+        [string] $environment = "",
+        [string] $apiVersion = "v2.15",
+        [string] $endPoint = ""
```

Comment:
For a helperfunction like this, should parameters be defaulted?
Shouldn't they be defaulted in the calling function and always transferred?

Reply:
I set defaults to empty strings.